### ISSUE TITLE
Unbalanced calls to begin/end appearance 

### ISSUE THREAD

**Issue by joergbirkhold** [CosmicMind/Motion#29]
I got this message when presenting this  Navigationcontroller.
"Unbalanced calls to begin/end appearance"
(The animation works as expected)
```swift
class AnimatableNavController: UINavigationController {

    var transtype:MotionTransitionAnimationType?
    
    open override func viewDidLoad() {
        super.viewDidLoad()
        isMotionEnabled = true
        motionTransitionType = transtype ?? .push(direction: .right)
        motionNavigationTransitionType = transtype ?? .push(direction: .right)
    }
}
```

**Comment by daniel-jonathan** [comment]
Hey! I am sorry it took so long to get back to you. I don't recall seeing this. Please, if you still have any issues, let me know. Thank you!

**Comment by StepanetsDmtry** [comment]
So this bug is still present. For example, I have NavigationDrawer controller with ToolBarController as root view controller. Then I try to present modally SearchBarController, motion animation worked perfectly

```
let searchController = AppStoryboard.Tree.viewController(viewControllerClass: SearchViewController.self)
let searchBarController = SearchBarController(rootViewController: searchController)
searchBarController.isMotionEnabled = true
searchBarController.motionTransitionType = .autoReverse(presenting: .fade)
toolbarController?.present(searchBarController, animated: true)
```

But in log I see messages: 
```
Unbalanced calls to begin/end appearance transitions for <Material.SearchBarController: 0x7f8db554bde0>.
Unbalanced calls to begin/end appearance transitions for <NavigationDrawerController: 0x7f8db680b000>.
```

**Comment by joergbirkhold** [comment]
Sorry for the delay too, but I also still see this.

**Comment by daniel-jonathan** [comment]
I will see if I can replicate this issue. If you have a sample project that replicates this 100% of the time, I would appreciate it. Thank you for sharing the issue.

**Comment by StepanetsDmtry** [comment]
Hello @danieldahan thank you for your fast reply. I have attached a sample project. 

[SampleMotion.zip](https://github.com/CosmicMind/Motion/files/2165065/SampleMotion.zip)

In this project current issue representing in 100% of the time.

**Comment by daniel-jonathan** [comment]
@OrkhanAlikhanov Let's take a look at this.

**Comment by daniel-jonathan** [comment]
This is now fixed in [Motion 1.4.3](https://github.com/CosmicMind/Motion/releases/tag/1.4.3). Thank you!

**Comment by StepanetsDmtry** [comment]
Thank you too! Great work!
